everything_else/reproducing.m

```diff
@@ -353,10 +353,15 @@
 b1 = b(:,1); b2 = b(:,2); b3 = b(:,3); b4 = b(:,4);
 
 f = b1 + 2*b1.*b2 - 4*b1.*b2.*b3 + 2*b1.*b2.*b3.*b4;
-
 f_sym = (1/2)*(f + (1 - f));
 f_anti = (1/2)*(f - (1 - f));
 
+f_new = f_sym + f_anti;
+
+LHS=min(reshape(f_new,[]));
```

Comment:
Hi,

So this commit is almost good, but I'd suggest you trying to run this code in the MatLab environment (https://matlab.mathworks.com/, should be free for UWaterloo ppl) and you'll discover that your reshape function is incorrectly used (take a look at the reference here: https://www.mathworks.com/help/matlab/ref/reshape.html). Let me know if you have any questions!